src/components/Autocomplete/SectionItemsList/SectionItemsList.tsx

```diff
@@ -14,36 +14,34 @@ type SectionItemsListProps = {
   key?: string;
 };
 
-// eslint-disable-next-line func-names
-const DefaultRenderSectionItemsList: RenderSectionItemsList = function ({ section }) {
-  const { getSectionProps } = useContext(CioAutocompleteContext);
+const getSectionTitle = (section: Section): string => {
   const { type, displayName } = section;
-  let sectionTitle = displayName;
 
-  if (!sectionTitle) {
-    switch (type) {
-      case 'recommendations':
-        sectionTitle = section.podId;
-        break;
-      case 'autocomplete':
-        sectionTitle = section.indexSectionName;
-        break;
-      case 'custom':
-        sectionTitle = section.displayName;
-        break;
-      default:
-        sectionTitle = section.indexSectionName;
-        break;
-    }
+  switch (type) {
+    case 'recommendations':
+      return section.podId;
+    case 'autocomplete':
+      return section.indexSectionName;
+    case 'custom':
+      return displayName;
+    default:
+      return section.indexSectionName;
   }
+};
+
+// eslint-disable-next-line func-names
+const DefaultRenderSectionItemsList: RenderSectionItemsList = function ({ section }) {
+  const { getSectionProps } = useContext(CioAutocompleteContext);
+  const { displayName } = section;
+  const sectionTitle = camelToStartCase(displayName || getSectionTitle(section));
```

Comment:
I think we don't want to run `displayName` through `camelToStartCase.` If the user set it to be camelcase, we should leave it as is